pages/papers/ruby-human-language.mdx

```diff
@@ -4,38 +4,34 @@ lang: en
 description: We will learn why Ruby it is easy for all.
 ---
 
-<div css={{ color: "red", fontStyle: "italic", fontSize: "2em" }}>
-  In Progress
-</div>
-
-**Tags:** _Ruby_, _Language_
+**Tags:** _Ruby, Language_
 
-**Audience Level:** _~~Beginer~~_ _All_
+**Audience Level:** _Beginer, All_
 
 ### Elevator Pitch (300 characters)
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You have 300 characters to sell your talk. This is known as the "elevator pitch". Make it as exciting and enticing as possible.
 </div>
 
-¿Es ruby un lenguaje difícil de entender? ¿Que pasaría si te digo que ruby es un lenguaje de humanos para humanos? ¿Sera **ruby** un lenguaje complejo para iniciarme en la programación? Entonces, hablaremos sobre _ruby_ en lenguaje _ruby_. Y te sentirás humano.
+Is ruby a difficult language to understand? What would happen if I told you that ruby is a human language for humans? Will ** ruby ** be a complex language to start programming? Then, we will talk about _ruby_ in _ruby_ language. And you will feel human.
 
 ### Description
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You should make the description of your talk as compelling and exciting as possible. Remember, you're selling both the organizers of the events to select your talk, as well as trying to convince attendees your talk is the one they should see.
 </div>
 
-El creador de ruby _Yukihiro "Matz" Matsumoto_ a menudo menciona que esta _"tratando de hacer que Ruby sea natural, no simple"_ complementando esta frase diciendo _"Ruby es simple en apariencia, pero complejo por dentro, como el cuerpo humano"_ nada mas cerca de la verdad, y no se equivoca al mencionar humano, siendo esta la mejor forma de describir la armonia de ruby.
+The creator of ruby _Yukihiro "Matz" Matsumoto_ often mentions that he is _"trying to make Ruby natural, not simple"_ complementing this sentence by saying _"Ruby is simple in appearance, but complex inside, like the human body"_ nothing closer to the truth, and do not make mistakes when mentioning human, this being the best way to describe the ruby harmony.
 
-Y es que con solo prestando atencion a los detalles y leyendo con calma las lineas de ruby aunque no sepa programar, entiendo exactamente lo que va a suceder, por ejemplo `5.times { print "Nos *encanta* Ruby -- ¡es fuera de serie!" }` y es que en Ruby todo es un objeto y se demuestra aplicando una accion `times` a un numero `5` es pura magia de ruby.
+And is that just by paying attention to the details and reading the ruby lines calmly but do not know how to program, I understand exactly what is going to happen, for example `5.times {print" We * love * Ruby - it's out serial!" }` and it is that in Ruby everything is an object and it is demonstrated by applying a`times` action to a number `5` it is pure ruby magic.
```

Comment:
And it's that just [...], even though I do not know how to program, I understand [...] It is pure Ruby magic.